.gitignore

```diff
@@ -131,3 +131,4 @@ tmp_cache
 
 # python-language-server / Rope
 .ropeproject
+Dockerfile.dev
```

Comment:
@Vaskivskyi 
ad 1. I am aware of this is not clear enough. But this is just workaround. In initial addition of integration user **must** provide `API_ID` and `API_KEY` and this is stored in `data` collection. So we don't need `API_ID` and `API_KEY` in `options`. 
But in case that user will change `ID` or `KEY` for some reason in his station's configuration, he needs to update this in integration. As there is no option, how to update `data`, I am providing an `options` flow to change these to correct values, otherwise user will need to delete the integration and start all over to install it.

ad 2. as for now, I want to write a tests for files that are currently not tested. If it will consume lot of time, I is good point to add them into .coveragerc - thanks.